### PR TITLE
Refactor zuora properties

### PR DESCRIPTION
**Pull request by jonflynng**
Refactors the names of API response properties to be more in-line with Zuora's naming conventions (specifically the conventions in their REST API). That is, we are changing the property `amount` to be named `price`, and the property `interval` to be named `billingPeriod`. Refer to some examples of Zuora rest API responses [here](https://github.com/guardian/price-migration-engine/blob/main/lambda/src/test/resources/GuardianWeekly/CappedPriceIncrease/Subscription.json) and [here](https://github.com/guardian/price-migration-engine/blob/main/lambda/src/test/resources/GuardianWeekly/CappedPriceIncrease/Catalogue.json).

PR to change on the backend is [here](https://github.com/guardian/members-data-api/pull/872).

### CODE REVIEW

File: client/components/mma/accountoverview/ContributionUpdateAmountForm.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ export const ContributionUpdateAmountForm = (
 	const currentContributionOptions = (contributionAmountsLookup[
 		props.mainPlan.currencyISO
 	] || contributionAmountsLookup.international)[
-		props.mainPlan.interval as ContributionInterval
+		props.mainPlan.billingPeriod as ContributionInterval
```
Comment (andrewHEguardian) [inline client/components/mma/accountoverview/ContributionUpdateAmountForm.tsx:164]: Similar to above comment, should ContributionInterval also change

**Review by andrewHEguardian — APPROVED** [review]
👍 looks good to me, worth discussing whether to rename the other functions and variables that reference these, could be in another PR

File: client/components/mma/accountoverview/ContributionUpdateAmountForm.tsx (hunk 1 above)
Comment (jonflynng) [reply client/components/mma/accountoverview/ContributionUpdateAmountForm.tsx:164]: yep Ill amend it, the only thing I was thinking of leaving is the `ContributionUpdateAmount` component, as we use the word `amount` in manage.theguardian.com for contributions

File: client/components/mma/accountoverview/ContributionUpdateAmount.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ export const ContributionUpdateAmount = (
 					},
 					{
 						title: `${capitalize(
-							augmentInterval(props.mainPlan.interval),
+							augmentInterval(props.mainPlan.billingPeriod),
```
[the remark this answers is not in the document]
Comment (jonflynng) [reply client/components/mma/accountoverview/ContributionUpdateAmount.tsx:75]: yep ill change this one.

**Comment by jonflynng** [comment]
[Rename object properties to be in-line with Zuora's naming](https://trello.com/c/ZfJMciLT/3737-rename-object-properties-to-be-in-line-with-zuoras-naming)

**Review by charleycampbell — APPROVED** [review]
Looks good 👍

**Pull request merged**